pkgs/by-name/ro/roomarranger/package.nix

```
@@ -0,0 +1,78 @@
+{
+  at-spi2-atk,
+  autoPatchelfHook,
+  cairo,
+  cups,
+  fetchurl,
+  gdk-pixbuf,
+  gtk3,
+  lib,
+  libGL,
+  qt6,
+  stdenv,
+  xorg,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "roomarranger";
+  version = "10.0.1";
+
+  src = fetchurl {
+    url = "https://f000.backblazeb2.com/file/rooarr/rooarr1001-linux64.tar.gz";
+    hash = "sha256-OwJSOfyTQinVKzrJftpFa5NN1kGweBezedpL2aE4LbE=";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    qt6.wrapQtAppsHook
+    (lib.getLib stdenv.cc.cc)
+  ];
+
+  buildInputs = [
+    at-spi2-atk
+    cairo
+    cups
+    gdk-pixbuf
+    gtk3
+    libGL
+    qt6.qt3d
+    qt6.qtbase
+    qt6.qtwayland
+    xorg.libX11
+    xorg.libxcb
+    xorg.xcbutilimage
+    xorg.xcbutilkeysyms
+    xorg.xcbutilrenderutil
```

```suggestion
    gtk3
    qt6.qt3d
    qt6.qtwayland
```

How did you come up with this list of libraries? Most of them don't seem to have any use. The package still builds and runs without them.

---

initially i tried to build this package outside of the nixpkgs repository, and i ran into many errors due to missing libraries. i've been really struggling with this whole qt setup. i'll check these libs again.